README.md

```diff
@@ -1,56 +1,53 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# SOLOVINO TV SHOWS
 
-> Description the project.
+> This project is a place where you can like and comment your favorites TV Shows.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript, CSS, HTML.
+- Jest, Babel.
+- Visual Studio Code.
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://diegomero.github.io/javascript-capstone/dist/)
```

Comment:
- [ ] [REQUIRED] Please deploy correct branch becouse your deployment link shows other branches website, your `dev` branch shows other.  Below the picture I show difference between live demo link and dev branch's web-site.

- ### Left `Dev` Branch - Right `Live Link`

![image](https://user-images.githubusercontent.com/63915024/183135050-970e4c1d-4c7d-4fa9-b68c-13cd0fb478a1.png)

Reply:
- [ ] Link updated and the correct branch is being deployed